src/app/common/resources/resources.js

```diff
@@ -49,31 +49,31 @@ angular.module('just.common')
             };
 
             return {
-                users: crud("/api/v1/users"),
-                user: crud("/api/v1/users/:id"),
+                users: crud(+"users"),
```

Comment:
Yes! Måste man ha ibland.

Jag fixar.

/christopher

On Thu, Apr 21, 2016 at 12:57 PM, Jacob Burenstam <notifications@github.com>
wrote:

> In src/app/common/resources/resources.js
> <https://github.com/justarrived/just-match-frontend/pull/43#discussion_r60529530>
> :
>
> > @@ -49,31 +49,31 @@ angular.module('just.common')
> >              };
> >
> >              return {
> > -                users: crud("/api/v1/users"),
> > -                user: crud("/api/v1/users/:id"),
> > +                users: crud(+"users"),
>
> Syntax error? ;)
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/justarrived/just-match-frontend/pull/43/files/2cd2a2d3e1a026a579687fd9d7ca2e8a29d48b81#r60529530>
>



-- 

*Christopher Borghoff*
Owner / System Architect

● Mobile (Thailand) +66 9 0969 1039
● Mobile (Sweden)  +46 8 5592 2645
● christopher.borghoff@se-production.com
● Skype: CBorghoff
● LinkedIn <https://th.linkedin.com/in/christopherborghoff>, Twitter
<https://twitter.com/sedigitalprod>

● South East Digital Production Co.,Ltd.
● S.S.P. Tower, 16th Floor, Soi Sukhumvit 63 (Ekamai), Khlong Tan Nuea,
Vadhana, Bangkok 10110
● www.se-production.com